Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml

```diff
@@ -403,6 +405,110 @@ tasks:
                       value:
                         simple: active
             iscontext: true
+    - condition:
```

Comment:
Does this only get back true if asset is dev and service owner(s) found?

Reply:
Added the dev and service owner conditions as well.
AWS Systems manager branch is opted when it is dev, has service owner, agent is active, when its InsecureOpenssh and Linux ubuntu.